### ISSUE TITLE
Sounds

### ISSUE THREAD

**Issue by Njol** [Njol/Skript#323]
including the possiblility to block the script until the sound has played completely

**Comment by joeuguce99** [comment]
Yes, and maybe having the possibility to access resource pack sounds rather than Bukkit's predefined sounds.
